pkgs/development/libraries/jemalloc/default.nix

```diff
@@ -48,10 +51,14 @@ stdenv.mkDerivation rec {
     autogen
     autoconf
     automake
+    git
```

Comment:
```suggestion
    gitMinimal
```
We probably don't need ssh support here, so we can reduce dependencies

Reply:
thx, updated!